subcommand/server-acl-init/command.go

```diff
@@ -457,7 +463,7 @@ func (c *Command) Run(args []string) int {
 		}
 	}
 
-	if c.flagCreateInjectToken {
+	if c.flagCreateInjectToken || c.flagEnableHealthChecksController {
```

Comment:
if this is renamed to something like "add health checks rules" I can add a comment stating that it will imply the creation of the inject token as well.